### PR TITLE
Add a log statement when Bugsnag has finished loading (can be disabled using disableLog)

### PR DESCRIPTION
**Pull request by loopj**

### CODE REVIEW

File: src/bugsnag.js [hunk 1]
```diff
@@ -1253,6 +1258,9 @@
     self.leaveBreadcrumb({ type: "navigation", name: "Bugsnag Loaded" });
   }
 
+  // Log that we've successfully loaded
+  log("JavaScript error reporting initialized (https://bugsnag.com)", false);
```
Comment (wordofchristian) [inline src/bugsnag.js]: Maybe we could also include some other diagnostic information, such as release stage and whether or not autonotify is enabled

**Comment by apexskier** [comment]
If this featured is added, please don't make it log by default. In general, I expect to not see anything in the console in production unless something has broken. I'd be okay with this being a feature that's enabled with an `enableStartupLog` option.

**Comment by bengourley** [comment]
@loopj did you still want this feature added?

Based on @apexskier's feedback I think we should at least provide a configuration option to switch off the log.

Let me know and I'll take it from here if so.

**Comment by loopj** [comment]
Closing for now - we can reassess in future

**Pull request closed without merging**